### PR TITLE
clamscan: add page

### LINKED ISSUE #2320: page request: clamscan
add documentation for [clamscan](https://linux.die.net/man/1/clamscan)

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

Closes #2320

### CODE REVIEW

File: pages/common/clamscan.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`{{command}} | clamscan -`
+
+- Specify a virus database file or directory of files:
+
+`clamscan --database {{path/to/database_file_or_directory}}`
+
+- Output only infected files:
+
+`clamscan --infected`
```
Comment (sbrl) [inline pages/common/clamscan.md]: What does this scan?

Comment (pxgamer) [reply pages/common/clamscan.md]: By default it scans the current directory if no file/directory are provided.

Comment (sbrl) [reply pages/common/clamscan.md]: Ah, I see. Maybe we could work that into this example somehow then? It doesn't seem obvious to me.

Comment (pxgamer) [reply pages/common/clamscan.md]: Perhaps something like:
> - Scan the current directory and output only infected files:

Comment (sbrl) [reply pages/common/clamscan.md]: Sounds good!

Comment (pxgamer) [reply pages/common/clamscan.md]: Sorry, been busy this weekend. Should be able to update this in a couple of hours. :+1:

Comment (pxgamer) [reply pages/common/clamscan.md]: Updated it @sbrl

**Pull request merged**